build/nuget-release.yml

```diff
@@ -0,0 +1,172 @@
+  # 'Package.Version.ManualTrigger' is added as queue-time variable on build in Azure DevOps
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration release /property:Version=$(Build.BuildNumber)'
+          - task: CopyFiles@2
+            displayName: 'Copy build artifacts'
+            inputs:
+              contents: '**/bin/**/*.nupkg'
+              targetFolder: '$(Pipeline.Workspace)/templates'
+              flattenFolders: true
+          - task: PublishPipelineArtifact@0
+            displayName: 'Publish build artifacts'
+            inputs:
+              targetPath: '$(Pipeline.Workspace)/templates'
+              artifactName: Templates
+
+  - stage: IntegrationTests
+    displayName: Integration Tests
+    dependsOn: Build
+    condition: succeeded()
+    jobs:
+      - job: RunIntegrationTests
+        displayName: 'Run integration tests'
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DownloadPipelineArtifact@2
+            displayName: 'Download build artifacts'
+            inputs:
+              artifact: 'Templates'
+              path: '$(Build.SourcesDirectory)/templates'
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version)) in web API template'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Install template'
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: '-i "templates/*.nupkg"'
+          - task: DotNetCoreCLI@2
+            displayName: 'Create new project from template'
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: 'arcus-webapi -n MyProject -o /MyProject'
+          - task: Docker@1
+            displayName: 'Build Docker image from newly created web API project from template'
+            inputs:
+              dockerFile: /MyProject/Dockerfile
+              imageName: 'myproject:$(Build.BuildId)'
+          - task: Docker@1
+            displayName: 'Run new web API project Docker image from template'
+            inputs:
+              command: 'Run an image'
+              imageName: 'myproject:$(Build.BuildId)'
+              containerName: 'myproject'
+              ports: '$(Http.Port):$(Http.Port)'
+              envVars: |
+                ARCUS_HTTP_PORT=$(Http.Port)
+          - task: qetza.replacetokens.replacetokens-task.replacetokens@3
```

Comment:
Looks like we can use this YAML template here, right?

https://github.com/arcus-azure/azure-devops-templates/blob/master/test/run-integration-tests.yml#L19